<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulador de Procesos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #333;
    color: #fff;
}

QLineEdit {
    background-color: #444;
    color: #fff;
    border: 1px solid #666;
    selection-background-color: #555;
}

QPushButton {
    background-color: #555;
    color: #fff;
    border: 1px solid #666;
    padding: 5px;
}

QCheckBox, QRadioButton {
    color: #fff;
}

QCheckBox::indicator, QRadioButton::indicator {
    background-color: #444;
    border: 1px solid #666;
}

QCheckBox::indicator:unchecked, QRadioButton::indicator:unchecked {
    background-color: #333;
}

QCheckBox::indicator:checked, QRadioButton::indicator:checked {
    background-color: #555;
    border: 1px solid #666;
}

QMenuBar {
    background-color: #333;
    color: #fff;
}

QMenuBar::item {
    background-color: #333;
    color: #fff;
}

QMenuBar::item:selected {
    background-color: #555;
}

QMenu {
    background-color: #444;
    color: #fff;
}

QMenu::item {
    background-color: #444;
    color: #fff;
}

QMenu::item:selected {
    background-color: #555;
}
QPushButton:hover {
    background-color: #666;
}

QPushButton:pressed {
    background-color: #777;
}

QFrame#separatorLine {
    background-color: #666;
    width: 2px;  /* Ajusta el ancho de la línea según tus preferencias */
    margin: 0 5px;  /* Ajusta los márgenes según tus preferencias */
}



</string>
  </property>
  <widget class="QSpinBox" name="no_procesos">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="no_procesos_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="no_procesos_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="no_procesos_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>%</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No. Procesos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operacion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiempo Estimado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>segundos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Proceso</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
